src/main/java/hudson/plugins/dimensionsscm/DimensionsAPICallback.java

```diff
@@ -0,0 +1,14 @@
+package hudson.plugins.dimensionsscm;
+
+import com.serena.dmclient.api.DimensionsConnection;
+import hudson.FilePath;
+
+import java.io.IOException;
+import java.util.Calendar;
+import java.util.TimeZone;
+
+interface DimensionsAPICallback {
+
+    boolean hasRepositoryBeenUpdated(DimensionsAPI dimensionsAPI, DimensionsConnection connection,
```

Comment:
It looks like you've only updated the "have there been any changes" method, and not the "what the changes were" method (the changelog). This means the build may be triggered correctly, but the part of the page that shows what changed will still be incorrect.